extensions/objects/templates/ListOfSetsOfHtmlStringsEditor.js

```diff
@@ -28,41 +28,44 @@ oppia.directive('listOfSetsOfHtmlStringsEditor', [
       controller: ['$scope', function($scope) {
         var errorMessage = '';
         $scope.selectedRank = '';
-        $scope.maxPrevIndex = 0;
+        $scope.maxPrevIndex = 1;
 
-        if (!$scope.value) {
-          $scope.value = [];
-        }
         $scope.initArgs = $scope.getInitArgs();
         $scope.choices = $scope.initArgs.choices;
 
+        $scope.value = [[]];
```

Comment:
I remember us talking about this line resetting the $scope.values whenever rule editor is opened? The items' order should be preserved.

![1](https://user-images.githubusercontent.com/8678373/42415778-e37ba372-8278-11e8-9eb3-4cf6eb9b6833.png)